package.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+set -e
+
+pip3 install --user -r requirements.txt
+pip3 install --user -r requirements_dev.txt
+
+# compile go tools
+/bin/bash scripts/setup.sh
+
+# package python code
+pyinstaller -D nebula-bench.spec
+
+# tar
+cd dist
```

Comment:
You need to go back to the directory where the user executed the command

Reply:
`cd` is used in shell context, it won't impact original user workspace.
just like `sh -c 'cd code;echo $PWD'`.